### PR TITLE
Add Press worker processes

### PR DESCRIPTION
**Pull request by pumazi**
This adds configuration for Celery worker processes that work for the Press service. The settings for this work are in #704

The first portion of this is in #703 . Feel free to close that if you can read this one given the moves and all.

### CODE REVIEW

File: roles/press_common/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+# +++
+# Prerequisites
+# +++
+
+- name: install system level dependencies
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
```
Comment (dtwilliamson) [inline roles/press_common/tasks/main.yml]: build tools? can't whatever be built once elsewhere and a resulting package installed instead?

File: roles/press_common/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  file:
+    path: "/var/cnx/apps"
+    state: directory
+    recurse: yes
+    owner: www-data
+
+- name: checkout codebase
+  become: yes
+  become_user: www-data
+  git:
```
Comment (dtwilliamson) [inline roles/press_common/tasks/main.yml]: See previous comment

File: roles/press_common/tasks/main.yml (hunk 1 above)
Comment (pumazi) [reply roles/press_common/tasks/main.yml]: This is existing logic that has been moved. I'm not opposed to discussing that at another time though. 😸

File: roles/press_worker/templates/etc/supervisor/conf.d/press_celery_workers.conf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{% for i in range(0, hostvars[inventory_hostname].press_worker_count|default(1), 1) %}
```
Comment (Dantemss) [inline roles/press_worker/templates/etc/supervisor/conf.d/press_celery_workers.conf]: Isn't a default of 1 a little small? Especially if the tasks can take a really long time.

Comment (pumazi) [reply roles/press_worker/templates/etc/supervisor/conf.d/press_celery_workers.conf]: None of the currently implemented task will take a long time. Shouldn't take but 1-2sec max.

**Pull request merged**